docs/source/faq.rst

```diff
@@ -494,3 +494,49 @@ There are two kinds of constrained optimizations, one with soft constraints and
 Soft constraints do not have to be satisfied, but an objective function is penalized if they are unsatisfied. On the other hand, hard constraints must be satisfied.
 
 Optuna is adopting the soft one and **DOES NOT** support the hard one. In other words, Optuna **DOES NOT** have built-in samplers for the hard constraints.
+
+
+Can I monitor trials and make them failed automatically when they are killed by a scheduler?
```

Comment:
Is the process killed by only a scheduler?

Reply:
Nice point. I updated the title like 65d0121.